src/screens/Sensors/index.tsx

```diff
@@ -52,7 +49,7 @@ export default function Sensors({navigation}: DrawerNavProps) {
       setRefreshing(false);
       const numberOfBluetoothSensors = await db
         .get('sensors')
-        .query(Q.where('type', 'bluetooth'))
+        ?.query(Q.where('type', 'bluetooth'))
```

Comment:
You can remove this too. I figured out why this is failing. We are calling `sensors` but the table name is `sensor` I must have missed it when updating the schema. But we also don't need this query because we are just console.logging it. So we can remove it I think.